client/post-editor/media-modal/index.jsx

```diff
@@ -161,35 +173,35 @@ export const EditorMediaModal = React.createClass( {
 			selectedCount = this.props.mediaLibrarySelectedItems.length;
 		}
 
-		const confirmMessage = this.translate(
+		const confirmMessage = this.props.translate(
 			'Are you sure you want to permanently delete this item?',
 			'Are you sure you want to permanently delete these items?',
 			{ count: selectedCount }
 		);
 
 		accept( confirmMessage, this.confirmDeleteMedia );
-	},
+	}
 
-	onAddMedia: function() {
-		PostStats.recordStat( 'media_explorer_upload' );
-		PostStats.recordEvent( 'Upload Media' );
-	},
+	onAddMedia = () => {
+		recordStat( 'media_explorer_upload' );
+		recordEvent( 'Upload Media' );
+	}
 
-	onAddAndEditImage: function() {
+	onAddAndEditImage = () => {
 		MediaActions.setLibrarySelectedItems( this.props.site.ID, [] );
 
 		this.props.setView( ModalViews.IMAGE_EDITOR );
-	},
+	}
```

Comment:
I suppose these need to be bound for the accept lib? Following our style with static props, could we add `;` here for these functions. This is an assignment statement.